yarn.lock

```diff
@@ -11150,58 +11173,58 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@jest/console@npm:^29.4.3":
-  version: 29.4.3
-  resolution: "@jest/console@npm:29.4.3"
+"@jest/console@npm:^29.5.0":
```

Comment:
I think you need to run `yarn` again to get an updated yarn.lock

Reply:
I have ran `yarn install` multiple times this morning and nothing has changed. I will look at this more.

Reply:
Everything is back to 29.4.3